core/src/com/projectkorra/projectkorra/firebending/FireBlast.java

```diff
@@ -148,6 +150,11 @@ public boolean checkLocation(Block block) {
 			this.remove();
 			return false;
 		}
+		for (Block b : GeneralMethods.getBlocksAroundPoint(block.getLocation(), 1.4)) {
+			if (!isSnow(b)) continue;
+
+			dryWetBlocks(b, this, ThreadLocalRandom.current().nextInt(4) == 0);
```

Comment:
code style so feel free to disregard but I feel like this should just be a if (isSnow) { dry }, no need for an early continue if that's all thats happening